lib/Importer/Caffe2.cpp

```diff
@@ -562,6 +562,21 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
+  if (typeName == "Clip") {
+    // Load the inputs:
+    auto in = getNodeValueOrCreateVariableByName(op.input(0));
+    auto cmin = (dict.count("min") && dict["min"]->has_f())
```

Comment:
nit: I know we aren't consistent here, but I think we should make a push to use actual types when it's not too burdensome. So please use `float` here and below.

Reply:
Can you explain a little bit more? Not clear which part? You mean change `auto` to `float`?

Reply:
Yeah sorry, please use float instead of auto